docs/source/models_new.rst

```diff
@@ -111,6 +111,21 @@ Box MAPs are reported on COCO
 
 .. include:: generated/detection_table.rst
 
+Keypoint detection
+------------------
```

Comment:
This has the same level of heading as the table above. Should it be instead at the same level as "classification", "detection", "segmentation" sections?

Reply:
I think we should split Object Detection, Instance Segmentation and Person Keypoint Detection. In that case yes, this Keypoint detection would be at the same level as "classification", "detection", "segmentation". 
- Classification
    - Table of all available Classification weights
 ...
- Object Detection
    - Table of all available detection weights
- Keypoint detection
    - Table of all available detection weights

Otherwise we can keep them in the same group but the relative hierarchy need to change:

- Object Detection, Instance Segmentation and Person Keypoint Detection
    - Object Detection
        - Table of all available detection weights
    - Keypoint detection
        - Table of all available detection weights

Reply:
Thanks for the input @jdsgomes , that sounds good.

The currently title is `Object Detection, Instance Segmentation and Person Keypoint Detection`. Do we actually have Instance Segmentation models ATM?

Reply:
I don't think so

Reply:
Iisn't mask rcnn instance segmentation?

Reply:
Thanks @oke-aditya  you're right. As you also properly noted offline, mask-rcnn has a custom  mask_map metric, so it would make sense for it to have its own table. Would you like to submit a PR to fix this?

Thinking about our layout again, I guess it could look like

```
Object Detection, Instance Segmentation and Person Keypoint Detection 
  Object Detection
     table
  Instance segmentation
     table
  Keypoint detection
     table
```

Reply:
yep, will send a PR :)